### PR TITLE
Add TLS support for AMQP-Kafka bridge

### PR DESCRIPTION
**Pull request by scholzj**
In master the AMQP-Kafka bridge doesn't work because the bridge doesn't use TLS and the right port when connecting to the router. 

This PR adds the annotation to the bridge deployment to generate the secret with TLS certificates and maps this secret as a volume. A second PR  EnMasseProject/amqp-kafka-bridge#75 adds TLS support for the bridge it self.

### CODE REVIEW

File: templates/include/amqp-kafka-bridge.jsonnet [hunk 1]
```diff
@@ -29,12 +31,66 @@ local common = import "common.jsonnet";
...
+              }
+            ],
+            "volumes": [
+              {
+                "name": certSecretName,
+                "secret": {
+                  "secretName": certSecretName
+                }
+              }
+             ]
```
Comment (ppatierno) [inline templates/include/amqp-kafka-bridge.jsonnet]: maybe we could try to extract a function like the `clientContainer` as before to be less verbose here ?

Comment (lulf) [reply templates/include/amqp-kafka-bridge.jsonnet]: IMHO, I'd say this is fine for now. We have an outstanding task to look at ksonnet and get a better structure of the jsonnet files in place soon. Although verbose, it's a bit easier to understand.

Comment (scholzj) [reply templates/include/amqp-kafka-bridge.jsonnet]: I probably took a bit of a shortcut here :-). I did a quick search and it seems that the clientContainer function from `common.jsonnet` is not used for anything anymore. So it should be easy to add some support for volume mounts into it.

Comment (scholzj) [reply templates/include/amqp-kafka-bridge.jsonnet]: I updated the PR so that it uses the function again. Have a look.

**Pull request merged**